ci-operator/config/openshift-qe/ocpqe-security-tools/openshift-qe-ocpqe-security-tools-main__dast.yaml

```diff
@@ -28,7 +28,7 @@ resources:
       memory: 200Mi
 tests:
 - as: security-first-half
-  cron: 0 16 */21 * *
+  cron: 0 16 3 * *
```

Comment:
Ok makes sense, First half running at 16 hr every 21 days and second half every 5 hour every 21 days. It does not  matters which half runs first as per __dast-4.18.yaml

This one at 16 hr 3rd day of the month. Till 4.19 comes in picture, doesn't main points to 4.18? If not , what runs does this file controls?

Reply:
Main points to [4.17](https://github.com/openshift/release/blob/ada4e492058dbdc080aa6e92a821ef0fd7f4fc1e/ci-operator/config/openshift-qe/ocpqe-security-tools/openshift-qe-ocpqe-security-tools-main__dast.yaml#L23) and instead of continuing to update the file every time I created a new file with the newest version separately